### PR TITLE
Issue #38

### PR DESCRIPTION
**Pull request by rgerm**
Removes not working menu options from viewer.
Removes applet and charts from viewer.
Removes eclipse specific files.
Cleans up gitignore files.

The movsim config files are still parsed twice for the viewer. TODO

### CODE REVIEW

File: core/pom.xml [hunk 1]
```diff
@@ -10,6 +10,11 @@
     <packaging>jar</packaging>
     <name>MovsimCore</name>
     <description>Movsim core simulation component.</description>
+    <properties>
```
Comment (akegermany) [inline core/pom.xml]: not needed, already defined in root pom.xml

File: core/src/main/java/org/movsim/simulator/Simulator.java [hunk 2]
```diff
@@ -232,14 +232,7 @@ public SimulationRunnable getSimulationRunnable() {
...
 
-    /**
-     * Load scenario from xml.
-     * 
-     * @param scenario
-     * @param path
-     * @throws JAXBException
-     * @throws SAXException
-     */
+    // TODO apprently the viewwer needs a second initialization to work. Investigate and remove!
```
Comment (akegermany) [inline core/src/main/java/org/movsim/simulator/Simulator.java]: please correct the typos

**Comment by rgerm** [comment]
Eclipse config files are untracked from git. Already existing local ones should stay untouched.
.gitignore is updated. Childs are removed.

**Comment by akegermany** [comment]
Thanks for the cleanup

**Pull request merged**